### ISSUE TITLE
SC 2.4.4 Link Purpose: WCAG2ICT poses substitutions for the Intent and not the SC, should be removed

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#104]
Since the SC text for 2.4.4 Link Purpose has no web terms or need for substitution, so the text talking about word substitutions should be removed. Elsewhere in the 2013 WCAG2ICT document, no word substitutions are put forth for the Understanding the SC Intent section's content. So this should be removed. Currently says: 
>This applies directly as written and as described in [Intent from Understanding Success Criterion 2.4.4](http://www.w3.org/TR/2013/NOTE-UNDERSTANDING-WCAG20-20130905/navigation-mechanisms-refs.html#navigation-mechanisms-refs-intent-head) (also provided below), replacing both “web page” and “page” with “non-web documents and software” in the Intent from Understanding Success Criterion 2.4.4.

Additionally, this substitution isn't grammatically correct in all cases in the Intent. I suspect this is a remnant of when we tried to do the word substitutions for the intent and abandoned it because of the work involved.

**Comment by maryjom** [comment]
There is also a suggested Intent word substitution in the Definition of Content that says:
>Within WCAG2ICT wherever “content” or “web content” appears in a success criterion or Intent it should be replaced with “content” using the definition above. 

The mention of word substitution in the Intent should also be removed.

**Comment by maryjom** [comment]
This has been incorporated into the editor's draft.
